docs/v2-system-addon/data_dictionary.md

```diff
@@ -147,6 +147,7 @@ Schema definitions/validations that can be used for tests can be found in `syste
 | `action` | [Required] Either `activity_stream_event`, `activity_stream_session`, or `activity_stream_performance`. | :one:
 | `addon_version` | [Required] The version of the Activity Stream addon. | :one:
 | `client_id` | [Required] An identifier for this client. | :one:
+| `card_type` | Either ("bookmark", "history", "trending", "pinned", "not_pinned") | :one:
```

Comment:
Let's add a qualifier, either `Optional` or `Required` to all the new fields. I guess all should be `Optional` since they're not available for the old versions.

Reply:
nit: `not_pinned` is not of the interest in this case, let's omit it and keep the payload compact.

Reply:
If we consider that the "card_type" for Topsite items is send for more than just click do we still want to omit the "not_pinned" attribute?

Reply:
I don't think the "not_pinned" attribute is necessary here. If we were to fetch all the "not_pinned" event records from top sites, we can just do `WHERE source = 'TOPSITES' and card_type <> 'pinned'`